codes/options/train/train_template.yml

```diff
@@ -131,6 +131,9 @@ train:
   lr_G: !!float 1e-4 # 2e-4 # starting lr_G #Test, default: 1e-4
   lr_D: !!float 1e-4 # 4e-4 # starting lr_D #Test, default: 1e-4
 
+  # ESRGAN-FS Augmentations
+  use_filters: true
```

Comment:
I feel like it should be called `use_frequency_separation` or something a bit more descriptive about what it's for. `use_filters` sounds like it could be an OTF thing.

Reply:
Changes made.